recipe-box-frontend/src/main/frontend/src/Pages/LoginPage/UserLogin.css

```diff
@@ -0,0 +1,110 @@
+.login-card {
+    width: 460px;
+    height: 715px;
+    padding: 50px;
+    background-color: white;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    display: flex;
+    flex-direction: column;
+
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+.close-button {
+    position: absolute;
+    top: 20px;
+    right: 20px;
+    background-color: transparent;
+    border-radius: 5px;
+    padding: 10px;
+    transition: background-color 0.3s ease;
+}
+.close-button:hover {
+    background-color: lightgray;
+}
+
+.login-logo {
+    margin: 100px auto 65px;
+    width: 350px;
+    height: auto;
+}
+.input-group {
+    width: 100%;
+    margin-bottom: 30px;
+    display: flex;
```

Comment:
The space(margin) between inputs and buttons is too big

Reply:
![Screenshot 2024-09-06 185028](https://github.com/user-attachments/assets/06d7476e-f9b6-414c-a65a-18c0d32f2b2e)

But here I used 55 and it looks about right

(I don't know if I pasted the image correctly)